torchmultimodal/diffusion_labs/models/adm_unet/adm.py

```diff
@@ -453,7 +479,7 @@ def adm_unet(
     time_embed_dim: int = 512,
```

Comment:
Do all of these parameters make sense here anymore? Since a user can just call ADMUnet if they want to change many defaults. Maybe just expose "embedding_name", "image_channels", and "predict_variance"

Reply:
I just mean, that if the purpose of this builder is just to house the defaults used in ADM then I don't think we should allow the user to override all of the defaults. If they need custom parameters they can use ADMUnet. If they want the ADM defaults then they can use this functions with just the three parameters exposed I mentioned. I'm also okay with moving these defaults to ADMUnet and dropping this builder.

Reply:
I see. Maybe let's do that in a follow-up then? I think dalle2_decoder is using a bunch of these params rn so making this change would have implications for that

Reply:
These defaults match dalle2 decoder. But you're right we should update dalle2 decoder then to be simpler.